src/main/java/org/wso2/carbon/connector/FileCopy.java

```diff
@@ -181,15 +181,16 @@ private boolean copyFile(String source, String destination, String filePattern,
      * @param opts        FileSystemOptions
      * @throws IOException
      */
```

Comment:
Shall we update the method parameters in the javadoc?

Reply:
Done